CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.11.0] - 2020-08-20
```

Comment:
For the future: For 0.x.x version when a new feature is added we should bump patch.

Reply:
You mean to release it as `0.10.3` instead? Ok, what are the criteria for a minor bump? My bad, I was thinking since it will fail some existing builds it was safer that way

Reply:
It's ok. Criteria is what server spec says. If it starts with `0` then for added/fix you bump patch, for changes/removals you bump minor.